src/Mobile.Bff.Shopping/Program.cs

```diff
@@ -1,6 +1,7 @@
 ﻿var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
+builder.AddBasicServiceDefaults();  // Default health checks 
```

Comment:
```suggestion
```

This method is already called by AddServiceDefaults();

Must call: `app.MapDefaultEndpoints();`